TestingScripts/CommonTasks/Components.py

```diff
@@ -38,88 +38,34 @@ def IgnoreForComp(compToFind):
 def PdbForComp(compToFind):
     for comp in Components:
         if comp.lower() == compToFind.lower():
-            if 'pdb' in Components[comp].keys():
-                return Components[comp]['pdb']
-            else:
-                return None
-
+            return Components[comp]['pdb']
+            
 def RepoForComp(compToFind):
-    for comp in Components:
-        if compToFind.lower() in Components[comp]['product'].lower():
-            if 'repo' in Components[comp].keys():
-                repo = Components[comp]['repo']
-                repoPath = os.getenv('SrcRoot')
-                for path in repo:
-                    repoPath = os.path.join(repoPath, path)
-                return repoPath
-            else:
-                repoPath = os.path.join(os.getenv('SrcRoot'),'imodel-native', 'iModelCore', compToFind)
-                return repoPath
-    
-def LogPathForComp(compToFind):
     for comp in Components:
         if comp.lower() == compToFind.lower():
-            exeName = ExeForComp(comp)
-            if exeName is None:
-                return None
-            base_path = os.path.join(os.getenv('OutRoot'), 'Winx64', 'build', 'RunGTest')
-            log_path = base_path
-            if 'special_path' in Components[comp].keys():
-                for path in Components[comp]['special_path']:
-                    log_path = os.path.join(log_path, path)
-            else:
-                log_path = os.path.join(log_path, comp+"-GTest")
-            log_path = os.path.join(log_path, exeName, 'run', 'logs', 'test.log')
-            return log_path
+            repoPath = os.path.join(os.getenv('SrcRoot'),'imodel-native', 'iModelCore', compToFind)
+            return repoPath
 
 def ExePathForComp(compToFind):
-    for comp in Components:
-        if compToFind.lower() in Components[comp]['product'].lower():
-            exeName = ExeForComp(comp)
-            if exeName is None:
-                return None
-            base_path = os.path.join('%OutRoot%Winx64', 'Product')
-            exe_path = base_path
-            if 'special_path' in Components[comp].keys():
-                for path in Components[comp]['special_path']:
-                    exe_path = os.path.join(exe_path, path)
-            else:
-                exe_path = os.path.join(exe_path, comp+"-GTest")
-            exe_path = os.path.join(exe_path, exeName+'.exe')
-            return exe_path
+    exeName = ExeForComp(compToFind)
+    exe_path = os.path.join('%OutRoot%Winx64', 'Product')
```

Comment:
does %outroot% actually expand into an env variable?

Reply:
yes it is an env variable

Reply:
![image](https://github.com/user-attachments/assets/d6e4ce87-7822-42ad-ad2b-bcf8a30a96f4)
What am I doing wrong here to get OutRoot to expand into an env variable?

Reply:
changed it to os.getenv()